diff --git a/chrome/browser/ui/webui/help/BUILD.gn b/chrome/browser/ui/webui/help/BUILD.gn
index 17d8d3e2832c4d572054a7c866690703e9fe87a2..995b51c03e85add198c61227c5b5f70e8a170cd9 100644
--- a/chrome/browser/ui/webui/help/BUILD.gn
+++ b/chrome/browser/ui/webui/help/BUILD.gn
@@ -2,12 +2,14 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/update/buildflags/buildflags.gni")  # Growser-161
 import("//chrome/common/features.gni")
 
 source_set("help") {
   public = [ "version_updater.h" ]
   public_deps = [ "//base" ]
   deps = [ "//chrome/browser/upgrade_detector" ]
+  deps += [ "//brave/browser/ui/webui/help" ]
 
   if (is_chromeos) {
     sources = [
@@ -49,7 +51,7 @@ source_set("help") {
   }
 
   if (is_win) {
-    if (is_chrome_branded) {
+    if (is_official_build && growser_enable_self_update) {  # Growser-161
       sources = [ "version_updater_win.cc" ]
       deps += [
         "//chrome/app:generated_resources_grit",
