diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index ca0ab3981e0cb852608643cfb46a9aacdb98df66..b8693bb46fc4e60b55d773f05a21571a2128f3d5 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -9,6 +9,7 @@ import {pageVisibility} from './page_visibility.js';
 import type {SettingsRoutes} from './router.js';
 import {Route, Router} from './router.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -302,6 +303,7 @@ function createRoutes(): SettingsRoutes {
         loadTimeData.getString('performancePageTitle'));
   }
 
+  addBraveRoutes(r);
   return r as unknown as SettingsRoutes;
 }
 
@@ -335,7 +337,7 @@ export let routes: SettingsRoutes = Router.getInstance().getRoutes();
 // Returns the "effective" route when at chrome://settings/.
 export function getTopLevelRoute(): Route {
   if (!loadTimeData.getBoolean('isGuest')) {
-    return routes.PEOPLE;
+    return routes.GET_STARTED;
   }
 
   let guestTopLevelRoute = routes.SEARCH;
