diff --git a/chrome/browser/ui/page_action/page_action_properties_provider.cc b/chrome/browser/ui/page_action/page_action_properties_provider.cc
index 4d9ec05d2d72cbec3d4c3d193c0864a4487e0076..65277f29b16ca57ccc6eac03e0be1234d0a3028c 100644
--- a/chrome/browser/ui/page_action/page_action_properties_provider.cc
+++ b/chrome/browser/ui/page_action/page_action_properties_provider.cc
@@ -227,6 +227,20 @@ constexpr auto kPageActionProperties = base::MakeFixedFlatMap<
             .element_identifier = kAutofillAddressPageActionElementId,
         },
     },
+    {
+        kActionShowPartitionedStorage,
+        {
+            .histogram_name = "PartitionedStorage",
+            .type = brave::kPartitionedStorageActionIconType,
+        },
+    },
+    {
+        kActionShowPsstIcon,
+        {
+           .histogram_name = "PsstIcon",
+           .type = brave::kPsstIconActionIconType,
+        },
+    },
     {
         kActionVirtualCardEnroll,
         {
