diff --git a/chrome/browser/net/profile_network_context_service.cc b/chrome/browser/net/profile_network_context_service.cc
index c9ae7f58abdc9e650501363db9da48231ffecc15..7bd4c1d27f74794f5bac0b6ebc8c79012b375972 100644
--- a/chrome/browser/net/profile_network_context_service.cc
+++ b/chrome/browser/net/profile_network_context_service.cc
@@ -655,7 +655,7 @@ void ProfileNetworkContextService::RegisterProfilePrefs(
                                 false);
   registry->RegisterListPref(prefs::kHSTSPolicyBypassList);
   registry->RegisterListPref(prefs::kCACertificates);
-  registry->RegisterListPref(prefs::kCACertificatesWithConstraints);
+  BRAVE_REGISTER_CA_CERTIFICATES_WITH_CONSTRAINTS_PREF
   registry->RegisterListPref(prefs::kCADistrustedCertificates);
   registry->RegisterListPref(prefs::kCAHintCertificates);
   // For information about whether to reset the HTTP Cache or not, defaults
@@ -746,6 +746,7 @@ network::mojom::CTPolicyPtr ProfileNetworkContextService::GetCTPolicy() {
   std::vector<std::string> excluded_spkis(
       TranslateStringArray(ct_excluded_spkis));
 
+  BRAVE_PROFILE_NETWORK_CONTEXT_SERVICE_GET_CT_POLICY
   return network::mojom::CTPolicy::New(std::move(excluded),
                                        std::move(excluded_spkis));
 }
@@ -1415,9 +1416,8 @@ void ProfileNetworkContextService::ConfigureNetworkContextParamsInternal(
     network_context_params->file_paths->cookie_database_name =
         base::FilePath(chrome::kCookieFilename);
 
-    g_browser_process->system_network_context_manager()
-        ->AddCookieEncryptionManagerToNetworkContextParams(
-            network_context_params);
+    BRAVE_ADD_COOKIE_ENCRYPTION_MANAGER_TO_NETWORK_CONTEXT_PARAMS(
+        network_context_params);
 
 #if BUILDFLAG(ENTERPRISE_CACHE_ENCRYPTION)
     if (enterprise_encryption::ShouldEncryptHttpCache(profile_->GetPrefs())) {
