diff --git a/chrome/browser/ui/startup/bad_flags_prompt.cc b/chrome/browser/ui/startup/bad_flags_prompt.cc
index ec4de18dce2482604b684a33f4ebc9d6a03b667e..5d415c3df90e8e647e2548c6f2a46ded656aba2d 100644
--- a/chrome/browser/ui/startup/bad_flags_prompt.cc
+++ b/chrome/browser/ui/startup/bad_flags_prompt.cc
@@ -15,6 +15,7 @@
 #include "base/files/file_path.h"
 #include "base/strings/utf_string_conversions.h"
 #include "base/trace_event/memory_dump_manager.h"
+#include "brave/components/growser_snap/snap.h"  // Growser-256
 #include "build/build_config.h"
 #include "cc/base/switches.h"
 #include "chrome/browser/infobars/simple_alert_infobar_creator.h"
@@ -266,6 +267,9 @@ void ShowBadFlagsPrompt(content::WebContents* web_contents) {
 // for flags which are not available in about:flags.
 #if !BUILDFLAG(IS_ANDROID)
   for (const char* flag : kBadFlags) {
+    if (growser::IsSnapPackagingFlag(flag)) {  // Growser-256
+      continue;
+    }
     if (base::CommandLine::ForCurrentProcess()->HasSwitch(flag)) {
       ShowBadFlagsInfoBar(web_contents, IDS_BAD_FLAGS_WARNING_MESSAGE, flag);
       return;
