diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 403d010318ed068fa8a9027c695665a2f57e2206..933c50262bca8a57f5799347043b05f4338df812 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4750,6 +4750,7 @@ const FeatureEntry::FeatureVariation kDictationVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
