diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 18be12aa26edf23b70ea2f540e35ad40a0a90cac..3dca649c9c8579b6b5ca4edcaf2fb48a36ac61af 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -558,6 +558,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
