diff --git a/chrome/browser/net/profile_network_context_service.cc b/chrome/browser/net/profile_network_context_service.cc
index 67be696b30045eb76c78cfaefec83bdf9ba580d3..f112919aaf7524a3cd801279a9fd9a2eca948f4b 100644
--- a/chrome/browser/net/profile_network_context_service.cc
+++ b/chrome/browser/net/profile_network_context_service.cc
@@ -660,7 +660,7 @@ void ProfileNetworkContextService::RegisterProfilePrefs(
                                 false);
   registry->RegisterListPref(prefs::kHSTSPolicyBypassList);
   registry->RegisterListPref(prefs::kCACertificates);
-  registry->RegisterListPref(prefs::kCACertificatesWithConstraints);
+  BRAVE_REGISTER_CA_CERTIFICATES_WITH_CONSTRAINTS_PREF
   registry->RegisterListPref(prefs::kCADistrustedCertificates);
   registry->RegisterListPref(prefs::kCAHintCertificates);
   // For information about whether to reset the HTTP Cache or not, defaults
@@ -746,6 +746,7 @@ network::mojom::CTPolicyPtr ProfileNetworkContextService::GetCTPolicy() {
   std::vector<std::string> excluded_spkis(
       TranslateStringArray(ct_excluded_spkis));
 
+  BRAVE_PROFILE_NETWORK_CONTEXT_SERVICE_GET_CT_POLICY
   return network::mojom::CTPolicy::New(std::move(excluded),
                                        std::move(excluded_spkis));
 }
@@ -1418,9 +1419,8 @@ void ProfileNetworkContextService::ConfigureNetworkContextParamsInternal(
     network_context_params->file_paths->cookie_database_name =
         base::FilePath(chrome::kCookieFilename);
 
-    g_browser_process->system_network_context_manager()
-        ->AddCookieEncryptionManagerToNetworkContextParams(
-            network_context_params);
+    BRAVE_ADD_COOKIE_ENCRYPTION_MANAGER_TO_NETWORK_CONTEXT_PARAMS(
+        network_context_params);
 
 #if BUILDFLAG(ENTERPRISE_CACHE_ENCRYPTION)
     if (enterprise_encryption::ShouldEncryptHttpCache(profile_->GetPrefs())) {
