diff --git a/chrome/installer/linux/debian/control.template b/chrome/installer/linux/debian/control.template
index 1d7dff3e689edee2018347eb1afccfa13601caf5..3b8d4ad05135fff58fa7b083522bae7a99adb9c3 100644
--- a/chrome/installer/linux/debian/control.template
+++ b/chrome/installer/linux/debian/control.template
@@ -1,12 +1,14 @@
-Source: @@PACKAGE-@@channel
+Source: @@package_and_channel
 Section: web
 Priority: optional
 Maintainer: @@MAINTNAME <@@MAINTMAIL>
 Build-Depends: dpkg-dev, devscripts, fakeroot, xz-utils
 Standards-Version: 3.8.0
 
-Package: @@PACKAGE-@@channel
+Package: @@package_and_channel
 Provides: @@deb_provides
+Replaces: @@deb_replaces
+Conflicts: @@deb_conflicts
 Pre-Depends: @@deb_pre_depends
 Depends: @@deb_depends
 Architecture: @@architecture
