diff --git a/chrome/browser/browser_resources.grd b/chrome/browser/browser_resources.grd
index 161313092c0e7c77a7b73e968dd0a5cf56cc45a8..9c86320f38197ffa5aaecd9c23c25ab95920d367 100644
--- a/chrome/browser/browser_resources.grd
+++ b/chrome/browser/browser_resources.grd
@@ -88,15 +88,6 @@
       </if>
     </structures>
     <includes>
-      <if expr="is_win or is_macosx or is_linux">
-        <!-- First Run audio resources -->
-        <include name="IDR_INTRO_SOUND_ALL_SET_FLAC" file="resources\intro\sounds\all_set.flac" type="BINDATA" />
-        <include name="IDR_INTRO_SOUND_AMBIENT_FLAC" file="resources\intro\sounds\ambient.flac" type="BINDATA" />
-        <include name="IDR_INTRO_SOUND_FEATURE_SHOWCASE_AMBIENT_FLAC" file="resources\intro\sounds\feature_showcase_ambient.flac" type="BINDATA" />
-        <include name="IDR_INTRO_SOUND_FEATURE_SHOWCASE_PROGRESS_FLAC" file="resources\intro\sounds\feature_showcase_progress.flac" type="BINDATA" />
-        <include name="IDR_INTRO_SOUND_LOGO_FLAC" file="resources\intro\sounds\logo.flac" type="BINDATA" />
-        <include name="IDR_INTRO_SOUND_WELCOME_BACK_FLAC" file="resources\intro\sounds\welcome_back.flac" type="BINDATA" />
-      </if>
       <if expr="enable_src_internal">
         <include name="IDR_APC_PROMPTS_JSON" file="../../internal/password_manager/apc_prompts.json" type="BINDATA" />
       </if>
