diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index e1bdd7210e6d0b378d60c3b89110e5cdfc0702f6..7a9c51c809fd96700bc58c6c5ede224c3f067ca9 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -440,7 +440,7 @@ Browser* Browser::Create(const CreateParams& params) {
   // should handle this; see e.g. crbug.com/40154317 and crbug.com/40798999.
   CHECK_EQ(CreationStatus::kOk, GetCreationStatusForProfile(params.profile));
 
-  std::unique_ptr<Browser> browser = base::WrapUnique(new Browser(params));
+  std::unique_ptr<Browser> browser = base::WrapUnique(new BraveBrowser(params));
   Browser* const browser_ptr = browser.get();
   BrowserManagerServiceFactory::GetForProfile(params.profile)
       ->AddBrowser(std::move(browser));
@@ -456,7 +456,7 @@ std::unique_ptr<Browser> Browser::DeprecatedCreateOwnedForTesting(
   // should handle this; see e.g. crbug.com/40154317 and crbug.com/40798999.
   CHECK_EQ(CreationStatus::kOk, GetCreationStatusForProfile(params.profile));
 
-  std::unique_ptr<Browser> browser = base::WrapUnique(new Browser(params));
+  std::unique_ptr<Browser> browser = base::WrapUnique(new BraveBrowser(params));
   BrowserManagerServiceFactory::GetForProfile(params.profile)
       ->AddBrowserForTesting(browser.get());
   return browser;
@@ -467,8 +467,8 @@ Browser::Browser(const CreateParams& params)
       profile_(params.profile),
       window_(nullptr),
       tab_strip_model_delegate_(
-          std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
-      tab_strip_model_(std::make_unique<TabStripModel>(
+          std::make_unique<chrome::BraveTabStripModelDelegate>(this)),
+      tab_strip_model_(std::make_unique<BraveTabStripModel>(
           tab_strip_model_delegate_.get(),
           params.profile,
           // Tab groups are disabled for app browsers.
