diff --git a/chrome/browser/ui/navigator/browser_navigator.cc b/chrome/browser/ui/navigator/browser_navigator.cc
index ed62443f55234f8a65779e5a5e801caed52e82da..259cf8d3574b94ef25370f2f78d9463115334bf1 100644
--- a/chrome/browser/ui/navigator/browser_navigator.cc
+++ b/chrome/browser/ui/navigator/browser_navigator.cc
@@ -616,6 +616,7 @@ base::WeakPtr<content::NavigationHandle> NavigateImpl(
   }
 #endif  // BUILDFLAG(IS_CHROMEOS)
 
+BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/40136163): Remove this code after we integrate with intent
   // handling.
