diff --git a/chrome/browser/ui/tabs/tab_strip_model.cc b/chrome/browser/ui/tabs/tab_strip_model.cc
index e45f2404bd018694d1e4a06bccfeac3d267cdf49..0ccc4cc2a685f4b4f2d3a5e12d5fd777bc6c6450 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.cc
+++ b/chrome/browser/ui/tabs/tab_strip_model.cc
@@ -183,7 +183,8 @@ std::unique_ptr<TabGroupModel> TabGroupModelFactory::Create() {
 
 DetachedTabCollection::DetachedTabCollection(
     std::variant<std::unique_ptr<tabs::TabGroupTabCollection>,
-                 std::unique_ptr<tabs::SplitTabCollection>> collection,
+                 std::unique_ptr<tabs::SplitTabCollection>,
+                 std::unique_ptr<tabs::TreeTabNodeTabCollection>> collection,
     std::optional<int> active_index,
     bool pinned)
     : collection_(std::move(collection)),
@@ -478,6 +479,7 @@ TabStripModel::DetachTabsAndCollectionsForInsertion(
 
   std::vector<tabs::TabInterface*> tab_interfaces =
       GetTabsAtIndices(tab_indices);
+  BRAVE_DETACH_TABS_AND_COLLECTIONS_FOR_INSERTION_PREPARE_TREE
 
   std::vector<std::variant<std::unique_ptr<DetachedTab>,
                            std::unique_ptr<DetachedTabCollection>>>
@@ -500,6 +502,7 @@ TabStripModel::DetachTabsAndCollectionsForInsertion(
     } else if (tab_interface->IsSplit()) {
       owned_tabs_and_collections.emplace_back(
           DetachSplitTabForInsertion(tab_interface->GetSplit().value()));
+    BRAVE_DETACH_TABS_AND_COLLECTIONS_FOR_INSERTION_TREE_NODE_BRANCH
     } else {
       owned_tabs_and_collections.emplace_back(DetachTabWithReasonAt(
           index, TabRemovedReason::kInsertedIntoOtherTabStrip,
@@ -1779,6 +1782,7 @@ void TabStripModel::AddTab(std::unique_ptr<tabs::TabModel> tab,
     // in the address bar and presses Alt+Enter. This allows for opening a new
     // Tab to quickly look up something. When this Tab is closed, the old one
     // is re-activated, not the next-adjacent.
+    BRAVE_TAB_STRIP_MODEL_ADD_TAB
     inherit_opener = true;
   }
   tabs::TabModel* tab_ptr = tab.get();
