diff --git a/chrome/browser/ui/views/profiles/first_run_flow_controller.cc b/chrome/browser/ui/views/profiles/first_run_flow_controller.cc
index deb8fb0dc354640e02cccf71f4423fb01108183a..562b11013dd28f2195b55fbd4ec582d4bb26fdfe 100644
--- a/chrome/browser/ui/views/profiles/first_run_flow_controller.cc
+++ b/chrome/browser/ui/views/profiles/first_run_flow_controller.cc
@@ -1031,23 +1031,6 @@ void FirstRunFlowController::Init() {
           content::GetAudioServiceStreamFactoryBinder());
     }
     if (sounds_manager_) {
-      sounds_manager_->Initialize(kLogoSoundKey, IDR_INTRO_SOUND_LOGO_FLAC,
-                                  media::AudioCodec::kFLAC, /*loop=*/false);
-      sounds_manager_->Initialize(kAmbientSoundKey,
-                                  IDR_INTRO_SOUND_AMBIENT_FLAC,
-                                  media::AudioCodec::kFLAC, /*loop=*/true);
-      sounds_manager_->Initialize(kWelcomeBackSoundKey,
-                                  IDR_INTRO_SOUND_WELCOME_BACK_FLAC,
-                                  media::AudioCodec::kFLAC, /*loop=*/false);
-      sounds_manager_->Initialize(kFeatureShowcaseAmbientSoundKey,
-                                  IDR_INTRO_SOUND_FEATURE_SHOWCASE_AMBIENT_FLAC,
-                                  media::AudioCodec::kFLAC, /*loop=*/true);
-      sounds_manager_->Initialize(
-          kFeatureShowcaseProgressSoundKey,
-          IDR_INTRO_SOUND_FEATURE_SHOWCASE_PROGRESS_FLAC,
-          media::AudioCodec::kFLAC, /*loop=*/false);
-      sounds_manager_->Initialize(kAllSetSoundKey, IDR_INTRO_SOUND_ALL_SET_FLAC,
-                                  media::AudioCodec::kFLAC, /*loop=*/false);
     }
   }
 
