diff --git a/components/omnibox/common/omnibox_feature_configs.cc b/components/omnibox/common/omnibox_feature_configs.cc
index fc14abacb1673..20aa99bcb4b10 100644
--- a/components/omnibox/common/omnibox_feature_configs.cc
+++ b/components/omnibox/common/omnibox_feature_configs.cc
@@ -375,7 +375,7 @@ DocumentProvider::~DocumentProvider() = default;
 
 BASE_FEATURE(AdjustOmniboxIndent::kAdjustOmniboxIndent,
              "AdjustOmniboxIndent",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 AdjustOmniboxIndent::AdjustOmniboxIndent() {
   const bool enabled = base::FeatureList::IsEnabled(kAdjustOmniboxIndent);
