diff --git a/components/enterprise/client_certificates/core/key_upload_client.cc b/components/enterprise/client_certificates/core/key_upload_client.cc
index 7d9e6aa72f44f103477da6e539a78ee3fbced5fb..34664cc1568e5faf22e599b7f34c403002e28e80 100644
--- a/components/enterprise/client_certificates/core/key_upload_client.cc
+++ b/components/enterprise/client_certificates/core/key_upload_client.cc
@@ -57,6 +57,7 @@ BPKUR::KeyType AlgorithmToType(
     case crypto::SignatureVerifier::RSA_PSS_SHA256:
       return BPKUR::RSA_KEY;
     case crypto::SignatureVerifier::ECDSA_SHA256:
+    case crypto::SignatureVerifier::ECDSA_SHA384:
       return BPKUR::EC_KEY;
   }
 }
