diff --git a/components/safe_browsing/core/browser/db/sb_protocol_manager_util.cc b/components/safe_browsing/core/browser/db/sb_protocol_manager_util.cc
index 4d5c202aa7cf48a50e6a109d3dd93e4dd1ce51bc..9a97b87799c8a48e3c39884fca9b95b217a944df 100644
--- a/components/safe_browsing/core/browser/db/sb_protocol_manager_util.cc
+++ b/components/safe_browsing/core/browser/db/sb_protocol_manager_util.cc
@@ -226,7 +226,7 @@ PlatformType GetCurrentPlatformType() {
 #elif BUILDFLAG(IS_MAC)
   return OSX_PLATFORM;
 #else
-  return ANDROID_PLATFORM;
+  return ANY_PLATFORM;  // Growser-303: see the recipe
 #endif
 }
 
