diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 3ea56d3d95d0d38e9a0f45a0ffa9014706d793c1..4f68bbad9a61cfcf4c78336d852779dcbbed26fa 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -520,6 +520,9 @@ static_library("browser") {
     ]
     deps += [ "//content/public/common" ]
   }
+  sources += brave_components_omnibox_browser_sources
+  public_deps += brave_components_omnibox_browser_public_deps
+  deps += brave_components_omnibox_browser_deps
 }
 
 source_set("pref_names") {
@@ -1104,6 +1107,7 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
 }
 
 mojom("mojo_bindings") {
+  generate_legacy_js_bindings = true
   sources = [ "omnibox.mojom" ]
   webui_module_path = "chrome://resources/mojo/components/omnibox/browser"
 
