diff --git a/components/on_device_translation/features.cc b/components/on_device_translation/features.cc
index 89c1ac1cc065a09bd6d24b9cb844d2a2ddbefb8f..5b6f2763ac36403c410b0377914135c8c3a7994b 100644
--- a/components/on_device_translation/features.cc
+++ b/components/on_device_translation/features.cc
@@ -38,8 +38,8 @@ const base::FeatureParam<base::TimeDelta> kTranslationAPIServiceIdleTimeout{
 const base::FeatureParam<size_t> kTranslationAPIMaxServiceCount{
     &blink::features::kTranslationAPI, "TranslationAPIMaxServiceCount", 10};
 
-BASE_FEATURE(kAutoDownloadTranslateLanguagePacks,
-             base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kAutoDownloadTranslateLanguagePacks,
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 const base::FeatureParam<std::string>
     kAutoDownloadTranslateLanguagePacksLanguagePairs{
