diff --git a/content/child/runtime_features.cc b/content/child/runtime_features.cc
index 95b2766f0a8b02bd0beb52cd46f1be94698f3acb..869e514e6074a8482ac248ab70cb7282eba2d44b 100644
--- a/content/child/runtime_features.cc
+++ b/content/child/runtime_features.cc
@@ -515,8 +515,7 @@ void ResolveInvalidConfigurations() {
         "FencedFramesLocalUnpartitionedDataAccess", false);
   }
 
-  if (!base::FeatureList::IsEnabled(
-          attribution_reporting::features::kConversionMeasurement)) {
+  if (false /* growser #109: keep attributionSrc exposed like Chrome */) {
     LOG_IF(WARNING, WebRuntimeFeatures::IsAttributionReportingEnabled())
         << "AttributionReporting cannot be enabled in this "
            "configuration. Use --"
