diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
index 23b69a1a1f6af9eb4902b38cedfa863fbc260b2c..3c470007ec61175634d0f2c8fdc3bd3eb83c3a17 100644
--- a/content/browser/web_contents/web_contents_impl.cc
+++ b/content/browser/web_contents/web_contents_impl.cc
@@ -5551,6 +5551,7 @@ FrameTree* WebContentsImpl::CreateNewWindow(
   // SiteInstance is created in a new BrowsingInstance.
   scoped_refptr<SiteInstance> site_instance;
   if (params.opener_suppressed) {
+    BRAVE_WEB_CONTENTS_IMPL_CREATE_NEW_WINDOW_OPENER_SUPPRESSED
     if (is_guest) {
       // For guests, noopener windows can be created in a new BrowsingInstance
       // as long as they preserve the guest's StoragePartition.
@@ -5648,7 +5649,7 @@ FrameTree* WebContentsImpl::CreateNewWindow(
 
   // Save the window for later if we're not suppressing the opener (since it
   // will be shown immediately).
-  if (!params.opener_suppressed) {
+  if (!params.opener_suppressed && !params.no_javascript_access) {
     if (!is_guest) {
       WebContentsView* new_view = new_contents_impl->view_.get();
 
