diff --git a/DEPS b/DEPS
index 4bac4e6ea3b422551b19eb783a1d14308419f835..a816f6968a2f9800462783477ceff89a5fbb824a 100644
--- a/DEPS
+++ b/DEPS
@@ -4438,7 +4438,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
