diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 9024abe3fe13d3395ec7c8d9a0ab8c4f7feba786..7a6062a001e5bfd4beabd260bc8ffc2de2cc2486 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -940,8 +940,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
