diff --git a/ios/chrome/browser/shared/model/url/url_util.mm b/ios/chrome/browser/shared/model/url/url_util.mm
index fa16702ba47909095f118ceaa9bafbffc37b2a60..5b301005d29157bda764c2a02eb6d7a12f3fa0ce 100644
--- a/ios/chrome/browser/shared/model/url/url_util.mm
+++ b/ios/chrome/browser/shared/model/url/url_util.mm
@@ -124,7 +124,7 @@ NSSet<NSString*>* GetItmsSchemes() {
                        // There's no evidence that itms-bookss is actually
                        // supported, but over-inclusion costs less than
                        // under-inclusion.
-                       @"itms-books", @"itms-bookss", @"itms-services",
+                       @"itms-books", @"itms-bookss",  // Growser-346: no itms-services
                        @"itms-beta", @"itms-betas", @"itms-watch",
                        @"itms-watchs", @"itms-podcasts", @"itms-podcastss",
                        nil];
