diff --git a/ios/chrome/browser/flags/about_flags.mm b/ios/chrome/browser/flags/about_flags.mm
index 8c0a3b1b43c68e734dcd0cd2040a73cfb4a0c3bb..e9ec06e37fca19d9edf0b789c0dbfb337f51a3cf 100644
--- a/ios/chrome/browser/flags/about_flags.mm
+++ b/ios/chrome/browser/flags/about_flags.mm
@@ -1342,6 +1342,7 @@ const FeatureEntry::FeatureVariation kCrossDeviceSigninVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 constexpr auto kFeatureEntries = std::to_array<flags_ui::FeatureEntry>({
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
     {"build-external-privacy-context",
      flag_descriptions::kBuildExternalPrivacyContextName,
      flag_descriptions::kBuildExternalPrivacyContextDescription,
