diff --git a/ios/chrome/browser/signin/ui/avatar/BUILD.gn b/ios/chrome/browser/signin/ui/avatar/BUILD.gn
index 24b0fdb6f8ba5be5a2a52db9a29888e03cd4d8c8..491204d219312d6425c50dc689f4ff8a99485bc2 100644
--- a/ios/chrome/browser/signin/ui/avatar/BUILD.gn
+++ b/ios/chrome/browser/signin/ui/avatar/BUILD.gn
@@ -11,6 +11,7 @@ source_set("avatar") {
   ]
   deps = [
     "//base",
+    "//ios/chrome/browser/providers/intelligence/signin",
     "//ios/chrome/browser/signin/model:constants",
     "//ios/public/provider/chrome/browser/intelligence/signin",
   ]
