diff --git a/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm b/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
index 9bcd4f88812f602181cbf8eaed9afcf35064de5a..d699d06d5c9ab2f10d53904c14c4f22c562e4b56 100644
--- a/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
+++ b/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
@@ -825,6 +825,7 @@ constexpr CGFloat kBatchUploadSymbolPointSize = 22.;
     case syncer::UserSelectableType::kSavedTabGroups:
     case syncer::UserSelectableType::kProductComparison:
     case syncer::UserSelectableType::kCookies:
+    case syncer::UserSelectableType::kAIChat:
       NOTREACHED();
   }
   DCHECK_NE(itemType, 0);
