diff --git a/ui/android/java/strings/translations/android_ui_strings_ru.xtb b/ui/android/java/strings/translations/android_ui_strings_ru.xtb
index 49b624fcdf01a7777cffb84b77654d60e3bc855e..a76942137211f0ac20d47ba2a1a7e97dc778f8f1 100644
--- a/ui/android/java/strings/translations/android_ui_strings_ru.xtb
+++ b/ui/android/java/strings/translations/android_ui_strings_ru.xtb
@@ -6,6 +6,7 @@
 <translation id="3240426699337459095">Ссылка скопирована</translation>
 <translation id="385051799172605136">Назад</translation>
 <translation id="4053720452172726777">Настройка и управление Google Chrome</translation>
+<translation id="7222463235177542287">Настройка и управление Growser</translation>
 <translation id="4177314441326667883">URL показан не полностью</translation>
 <translation id="4918844574251943176">Изображение скопировано.</translation>
 <translation id="6142675540468977324">URL показан полностью</translation>
